tests/eigen_svd/svd_cdouble_avx512.cpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file
+ *
+ * @copyright
+ * Copyright 2022 Intel Corporation.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * @test @b eigen_svd_cdouble
+ * @parblock
+ * This piece of code aims to stress test the FMA execution units of
```

Comment:
Maybe we should mention that this is an AVX_512 version of eigen_svd_cdouble.  I guess it's obvious (once you've fixed the name) but probably worth spelling out.